src/NotesField.php

```diff
@@ -10,14 +10,66 @@ class NotesField extends Field
     public $showOnCreation = false;
     public $showOnIndex = false;
 
+    public $showAddNote = true;
+
+    /**
+     * NovaNotesField constructor.
+     *
+     * @param $name
+     * @param null $attribute
+     * @param callable|null $resolveCallback
+     */
+    public function __construct($name, $attribute = null, callable $resolveCallback = null)
+    {
+        parent::__construct($name, $attribute, $resolveCallback);
+
+        $this->withMeta([
+            'showAddNote' => $this->showAddNote,
+        ]);
+    }
+
     /**
      * Sets the placeholder value displayed on the field.
      *
      * @param string $placeholder
-     * @return \OptimistDigital\NovaNotesField\NotesField
+     * @return NotesField
      **/
     public function placeholder($placeholder)
     {
         return $this->withMeta(['placeholder' => $placeholder]);
     }
+
+    /**
+     * Sets the input size.
+     *
+     * @param string $inputWidth
+     * @return NotesField
+     **/
+    public function inputWidth($inputWidth)
+    {
+        return $this->withMeta(['inputWidth' => $inputWidth]);
+    }
+
+    /**
+     * Show or hide the AddNote input.
+     *
+     * @param string $showAddNote
+     * @return NotesField
+     **/
+    public function showAddNote($showAddNote)
+    {
+        return $this->withMeta(['showAddNote' => $showAddNote]);
+    }
+
+    /**
+     * Show or hide the notes section
+     *
+     * @param string $showNotes
+     * @return NotesField
+     **/
+    public function showNotes($showNotes)
+    {
+        return $this->withMeta(['showNotes' => $showNotes]);
```

Comment:
This doesn't seem to be used in the JS either. What should it do, hide the notes field altogether?

Reply:
Sorry, I was playing with the idea of having an option to hide notes for roles/permissions that do not have permission to view them. Meant to remove this before pushing.

I can put it back in if you think it's useful.